nix/flakebox.nix

```diff
@@ -979,19 +979,16 @@ in
       args = "--package fedimint-client --package fedimint-client-wasm --package fedimint-wasm-tests --no-default-features";
     };
 
-    cargoWorkspacesCheckDefaultFeatures = cargoWorkspacesCommand {
-      cmd = "cargo check";
-      pnameSuffix = "default-feats";
+    cargoWorkspacesCheckDefaultFeatures = cargoCheckCommand {
+      args = "--workspace";
```

Comment:
Are we supposed to be adding `--workspace` to `cargo workspaces exec ...`? I don't think so. I was testing without it locally and it looked as it should...

Reply:
But initial local testing vs regressions is an early veto for this approach

Reply:
The pain point of resolving the bottleneck for untangling issues when publishing to `crates.io` is resolved as long as https://github.com/fedimint/fedimint/pull/8021 behaves well in the merge queue, so the extra optimizations probably aren't worth pursuing for now.

Reply:
> Nah, that's probably a bit rowdy to add `--workspace` to `cargo workspaces exec`, but I'm swapping out `cargoWorkspacesCommand` for `cargoCheckCommand` so this is actually `cargo check --workspace`.

I'm a bit confused. The whole thing is there to build each individual crate with certain features, because it's the only way to guarantee they they individually work. It is possible, and even somewhat likely that certain package builds together  as a workspace, because other crates enable stuff that it itself is missing. So I think we need to try them one by one to be actually sure.

Building whole workspace with all features enabled / disabled should be somewhat faster, but it will only give us ... donno... 50% of the same confidence that everything is OK.